common_ci_utils/random_utils.py

```diff
@@ -86,11 +86,11 @@ def _parse_dd_size_to_kb(size):
         unit = size[-1]
         size = int(size[:-1])
         if unit == "K":
-            return size
-        if unit == "M":
             return size * 1024
-        if unit == "G":
+        if unit == "M":
             return size * 1024 * 1024
+        if unit == "G":
+            return size * 1024 * 1024 * 1024
```

Comment:
above function return in KB, so I didn't see anything wrong here.

Reply:
either you need to append K to bs size or chnage the exiting function name and other variables to reflect the new chnages